packaging/cloudify-agents.spec

```diff
@@ -0,0 +1,45 @@
+%define _agents_dir /opt/manager/resources/packages/agents
+
+Name:           cloudify-agents
+Version:        %{CLOUDIFY_VERSION}
+Release:        %{CLOUDIFY_PACKAGE_RELEASE}%{?dist}
+Summary:        Cloudify's agents bundle
+Group:          Applications/Multimedia
+License:        Apache 2.0
+URL:            https://github.com/cloudify-cosmo/cloudify-agent
+Vendor:         Cloudify Platform Ltd.
+Packager:       Cloudify Platform Ltd.
+
+BuildRequires:  python >= 2.7
+Requires(pre):  shadow-utils
+
+
+%description
+Cloudify Agent packages
+
+
+%prep
+curl https://raw.githubusercontent.com/cloudify-cosmo/cloudify-versions/master/packages-urls/agent-packages.yaml > /tmp/agents-list.txt
```

Comment:
If you don't like `curl` here it should be possible to specify this URL as `Source0:` and have `build_rpm.py` download it to ${RPM_SOURCE_DIR} for you

Reply:
& you can then refer to it as `%{S:0}` (see cloudify-premium's spec for an example)

Reply:
I PRd your PR https://github.com/cloudify-cosmo/cloudify-manager/pull/1146